src/Resources/views/CRUD/Association/list_many_to_one.html.twig

```diff
@@ -14,10 +14,11 @@ file that was distributed with this source code.
 {% block field %}
     {% if value %}
         {% set route_name = field_description.options.route.name %}
-        {% if not field_description.options.identifier|default(false) and
-              field_description.hasAssociationAdmin and
-              field_description.associationadmin.hasRoute(route_name) and
-              field_description.associationadmin.hasAccess(route_name, value)
+        {% if not field_description.options.identifier|default(false)
+            and field_description.hasAssociationAdmin
+            and field_description.associationadmin.hasRoute(route_name)
+            and field_description.associationadmin.hasAccess(route_name, value)
+            and value.id
```

Comment:
Are we sure about this one?, what if my entity does not have a field called id, maybe I could use another name for my primary key, and I also could have a composite key.

Not sure what is the correct fix for this, but I am not sure about this one

Reply:
:thinking: makes sense… I think we have an admin method for this, don't we? We probably do, since there is one for every thing :trollface:

Reply:
Didn't think this through enough.

Reply:
`field_description.associationadmin.id(value)` this should work, we use that in templates, will test later and try this one more time 😄